tljh/requirements-base.txt

```diff
@@ -5,8 +5,10 @@
 jupyterhub==1.2.*
 notebook==6.1.*
 # Install additional notebook frontends!
-jupyterlab==2.*
+jupyterlab==3.*
 nteract-on-jupyter==2.1.*
+# Install jupyterlab extensions from PyPI
+jupyterlab-widgets==1.*
```

Comment:
Maybe worth updating to `ipywidgets==7.6.*` directly, which now depends on `jupyterlab_widgets`?

Reply:
Which means that the explicit dependency on `jupyterlab-widgets` would not be needed with `ipywidgets>=7.6`